packages/valory/skills/trader_abci/composition.py

```diff
@@ -76,6 +77,7 @@
     FinishedMarketManagerRound: SamplingRound,
     FailedMarketManagerRound: ResetAndPauseRound,
     FinishedDecisionMakerRound: RandomnessTransactionSubmissionRound,
+    RefillRequiredRound: ResetAndPauseRound,
```

Comment:
Before this change where were we transitioning to after the `RefillRequiredRound` round?

Reply:
nowhere, it was missing